src/page/sign-in-step.page.ts

```diff
@@ -0,0 +1,25 @@
+import { $, ElementFinder } from 'protractor';
+
+export class SignInStepPage {
+  private emailField: ElementFinder;
+  private passwdField: ElementFinder;
+  private loginSubmitButton: ElementFinder;
+
+  constructor () {
+    this.emailField = $('#email');
+    this.passwdField = $('#passwd');
+    this.loginSubmitButton = $('#SubmitLogin > span');
+  }
+
+  public async sendEmailField(value: string): Promise<void> {
+    await this.emailField.sendKeys(value);
+  }
+
+  public async sendPasswdField(value: string): Promise<void> {
```

Comment:
review my other comment about the `login` function, but `value` as the name for this param is too generic

Reply:
Done